src/Nethermind/Nethermind.Core/Extensions/Bytes.cs

```diff
@@ -334,6 +334,26 @@ public static byte[] Concat(byte[] bytes, byte suffix)
             return result;
         }
 
+        public static byte[] Concat(ReadOnlySpan<byte[]> bytes)
```

Comment:
Not used

Reply:
It is being used here: https://github.com/NethermindEth/nethermind/blob/a237b743886d6e67e0e2cc204298eaaf0e60c0f7/src/Nethermind/Nethermind.Core.Test/Builders/TestExecutionRequest.cs#L21

Reply:
You need to call .AsSpan() otherwise it will use an older method signature